### ISSUE TITLE
Dresser & Snick's Challenge room doesn't work

### ISSUE THREAD

**Issue by Bugydigital** [burnedpopcorn/PT-Chefs-Kiss-Web-Port#1]
it just crashes whenever I try to use the dresser and same with snick's challenge(this is on chromebook btw)
![Screenshot 2024-09-18 11 18 57 AM](https://github.com/user-attachments/assets/152de56e-1636-421e-8762-5c6fc60a0553)

**Comment by burnedpopcorn** [comment]
Yeah, the code is really fucked up, so I'm probably gonna discontinue this port
